specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/guestconfiguration_NotImplemented.json

```diff
@@ -0,0 +1,90 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "2018-11-20",
+      "title": "GuestConfiguration"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/providers/Microsoft.GuestConfiguration/guestConfigurationAssignments/{guestConfigurationAssignmentName}": {
+        "delete": {
```

Comment:
Please plan to implement delete as soon as possible. A resource that can be PUT but not deleted cannot be cleaned up

Reply:
We will do. Esha has started to work on it.

Reply:
Added specs for delete now. Esha is about to deploy the delete API as well ?